components/HomeHeader.vue

```diff
@@ -0,0 +1,160 @@
+<template>
+  <swiper class="swiper" :options="swiperOptions">
+    <swiper-slide v-for="item in items" :key="item.Id">
+      <div
+        class="slide-backdrop"
+        :style="{
+          backgroundImage: `url('${getBackdrop(item)}')`
+        }"
+      />
+      <div class="slide-backdrop-overlay" />
+      <div class="slide-content">
+        <v-container class="mx-10 mt-5">
+          <v-row>
+            <v-col cols="5">
+              <v-img
+                v-if="item.ImageTags && item.ImageTags.Logo"
+                :src="getLogo(item)"
+              />
+              <h1
+                v-else-if="item.Type === 'Episode'"
+                class="text-h2 text-truncate mb-2"
+              >
+                {{ item.SeriesName }}
+              </h1>
+              <h1 v-else class="text-truncate">{{ item.Name }}</h1>
+              <p
+                v-if="item.Type === 'Episode'"
+                class="mb-n1 text-truncate text-subtitle-2"
+              >
+                {{ item.SeasonName }}
+                {{ $t('episodeNumber', { episodeNumber: item.IndexNumber }) }}
+              </p>
+              <h2 v-else-if="item.Taglines" class="text-truncate">
+                {{ item.Taglines[0] }}
+              </h2>
+              <h2 v-if="item.Type === 'Episode'" class="text-h4 text-truncate">
+                {{ item.Name }}
+              </h2>
+              <media-info :item="item" year runtime rating class="mt-2" />
+              <p class="mt-2" v-html="getOverview(item)" />
+              <v-btn
+                class="mr-2"
+                color="primary"
+                min-width="8em"
+                depressed
+                rounded
+                :to="`./${item.Id}/play`"
```

Comment:
This redirects to `http://127.0.0.1:3000/4f036c78-e69a-7350-74e6-cd5a387c6433/play` 
`This page could not be found`

It probably should be `http://127.0.0.1:3000/item/4f036c78-e69a-7350-74e6-cd5a387c6433/play`